openslides/assignment/models.py

```diff
@@ -15,24 +15,36 @@
 from openslides.utils.exceptions import OpenSlidesError
 from openslides.utils.models import AbsoluteUrlMixin
 from openslides.utils.rest_api import RESTModelMixin
-from openslides.utils.utils import html_strong
 from openslides.users.models import User
 
 
-class AssignmentCandidate(RESTModelMixin, models.Model):
+class AssignmentRelatedUser(RESTModelMixin, models.Model):
     """
-    Many2Many table between an assignment and the candidates.
+    Many to Many table between an assignment and user.
     """
-    assignment = models.ForeignKey("Assignment")
-    person = models.ForeignKey(User, db_index=True)
-    elected = models.BooleanField(default=False)
-    blocked = models.BooleanField(default=False)
+    STATUS_CANDIDATE = 1
+    STATUS_ELECTED = 2
+    STATUS_BLOCKED = 3
+    STATUSES = (
+        (STATUS_CANDIDATE, ugettext_lazy('candidate')),
+        (STATUS_ELECTED, ugettext_lazy('elected')),
+        (STATUS_BLOCKED, ugettext_lazy('blocked')),
+    )
+
+    assignment = models.ForeignKey(
+        "Assignment",
+        db_index=True,
+        related_name='assignment_related_users')
+    user = models.ForeignKey(User, db_index=True)
+    status = models.IntegerField(
+        choices=STATUSES,
+        default=STATUS_CANDIDATE)
 
     class Meta:
-        unique_together = ("assignment", "person")
+        unique_together = ('assignment', 'user')
 
     def __str__(self):
-        return str(self.person)
+        return "%s <-> %s" % (self.assignment, self.user)
```

Comment:
I disagree. There is no harm in using double quotes. There is also no harm when the quotes are different on any string. This coding standard brings on improvment.

Also, of cause all my code is in single quotes, when I starting with this change yet. With your logic we could never change anything.

Recently you changed the style that we do not end a tuple with a comma. I don't like this neither but there is also no harm that you do this (I will not).
https://github.com/OpenSlides/OpenSlides/blob/master/openslides/users/serializers.py#L21